server/src/language/models/cache.js

```diff
@@ -120,6 +127,7 @@ export default class Cache {
       ...this.subroutines.filter(def => def.name.toUpperCase() === name),
       ...this.variables.filter(def => def.name.toUpperCase() === name),
       ...this.indicators.filter(def => def.name.toUpperCase() === name),
+      ...this.cursor.filter(def => def.name.toUpperCase() === name),
```

Comment:
Looks like you've updated the `cursors` property but missed renaming it elsewhere.